.NET/Microsoft.Recognizers.Text.DataTypes.DataDrivenTests/TestTimexHelpers.cs

```diff
@@ -93,7 +93,7 @@ public void DataTypes_Helpers_TimeRangeFromTimex()
         public void DataTypes_Helpers_DateFromTimex()
         {
             var timex = new TimexProperty("2017-09-27");
-            var date = TimexHelpers.DateFromTimex(timex);
+            var date = TimexHelpers.DateTimeFromTimex(timex);
```

Comment:
Both should return the same results, but the first one is "more correct" as the timex represents a date, not a datetime.